### ISSUE TITLE
comma after selector does not report an error

### ISSUE THREAD

**Issue by linxz** [stylelint/stylelint#4400]
> Clearly describe the bug

A comma after a selector does not report an error.

![image](https://user-images.githubusercontent.com/1132815/68352868-c0108980-0142-11ea-8530-708514c51ae0.png)

![image](https://user-images.githubusercontent.com/1132815/68352936-f4844580-0142-11ea-824b-9ebb0725404e.png)

> What CSS is needed to reproduce the bug?

```css
.a,
.b, {
  background-color: #323df3;
}
```

> What stylelint configuration is needed to reproduce the bug?

default configuration in https://stylelint.io/demo

**Comment by fanich37** [comment]
I don't think ```stylelint``` checks the validity of your css.

**Comment by linxz** [comment]
I also agree with you. thanks.

**Comment by jeddy3** [comment]
You can use [`csstree-validator`](https://www.npmjs.com/package/csstree-validator) for this. Alternatively, you can [resurrect the old `selector-no-empty`](https://github.com/stylelint/stylelint/pull/1977/files) rule as [a plugin](https://github.com/stylelint/stylelint/blob/master/docs/developer-guide/plugins.md).
